brands/shop/convenience.json

```diff
@@ -507,6 +507,16 @@
       "shop": "convenience"
     }
   },
+  "shop/convenience|Dari Mart": {
+    "tags": {
+      "brand": "Dari Mart",
+      "brand:website": "https://www.darimart.com/",
```

Comment:
```suggestion
```

Reply:
We haven't been including website tags with other entries so I recommend removing this.

Reply:
It seems like a perfect way to add some extra data to the objects, but I think following convention is more important, so I've removed the line.

Reply:
Agreed.  I’d say this is worth discussing in the #poi channel of the OSM-US Slack.